app/css/animator.css

```diff
@@ -53,6 +53,47 @@ a {
   text-align: center;
 }
 
+/* ========== CONFIRM BOXES ========== */
+
+#confirm-container {
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    background-color: rgba(23, 23, 23, 0.5);
+    z-index: 4;
+    text-align: center;
+}
+
+#confirm-container.hidden { display: none; }
+
+#confirm {
+    position: relative;
+    top: calc(50vh - 10em);
+    display: inline-block;
+    width: 20em;
+    background: #000;
+    padding: 0 1em 1em 1em;
+    border: solid 1px #D9D9D9;
+}
+
+#confirm-title {
+    font-size: 1.2em;
+    color: rgb(224, 224, 100);
+}
+
+#confirm-text { text-align: left; }
+
+#confirm-button-container{ float: right;}
```

Comment:
(I know, I am starting late. You have no idea.) Nit: Missing space before opening brace and after the semicolon.